samples/sampler/stories/default/WizardPanels.js

```diff
@@ -39,9 +39,14 @@ export const _WizardPanels = () => (
 			title="WizardPanel View 1"
 			prevButton={
 				boolean('custom first Panel prevButton', Config) ? (
-					<Button icon="closex" aria-label="exit">
-						Exit
-					</Button>
+					<Button icon="closex" aria-label="exit" />
+				) : (
+					void 0
+				)
+			}
+			prevButtonLabel={
+				boolean('custom first Panel prevButton', Config) ? (
```

Comment:
Shouldn't it be written in string form?
boolean('prevButtonLabel', Config)

I am not sure how to use this prop by looking at this sample.

Reply:
Well, putting new props for both labels would be better, maybe?